frontend/.babelrc

```diff
@@ -3,6 +3,6 @@
   "plugins": [
     ["@babel/plugin-transform-runtime", { "regenerator": true }],
     ["@babel/plugin-proposal-decorators", { "legacy": true }],
-    ["@babel/plugin-proposal-class-properties", { "loose": true }]
+    ["@babel/plugin-proposal-class-properties", { "loose": true }, "plugin-proposal-class-properties-alias"]
```

Comment:
I'm not sure if we want this change merged, but I needed it in my environment for the frontend to build successfully.

Reply:
It would be better to merge the fix PR before we merge this PR.